hicexplorer/hicPlotMatrix.py

```diff
@@ -117,8 +114,8 @@ def parse_arguments(args=None):
                         type=int,
                         default=72)
 
-    parser.add_argument('--pca',
-                        help='List of eigenvector from pca analysis as bigwig or bedgraph files.',
+    parser.add_argument('--bigwig',
+                        help='List of bigwig files to plot below the matrix',
                         type=str,
                         default=None,
                         nargs='+')
```

Comment:
We should change it to '1' because we only support one bigwig data track in the moment.